run.py

```diff
@@ -284,6 +401,10 @@ def generate_activity_plans(
         generating warm start activities based on the base year input data in
         order to generate "warm start" skims.
     """
+    
+    
+    if demand_model == 'polaris':
+        pilates.polaris.travel_model.run_polaris(forecast_year, settings, warm_start=True)
```

Comment:
prob missing an `else` statement here too. as written, if `demand_model == 'polaris'`, polaris will run and then activitysim will run as well